doc/source/building/working_with_images/vagrant_setup.rst

```diff
@@ -143,18 +175,19 @@ the following changes are required:
 
 
 An image built with the above setup creates a box file with the
-extension :file:`.vagrant.libvirt.box`. That box file can now be
-added to vagrant with the command:
+extension :file:`.vagrant.libvirt.box` or
+:file:`.vagrant.virtualbox.box`. That box file can now be added to vagrant
+with the command:
 
 .. code:: bash
 
    vagrant box add my-box image-file.vagrant.libvirt.box
 
 .. note::
 
-   Using the box requires a correct Vagrant installation on your machine,
-   the plugin ``vagrant-libvirt`` to be installed and the libvirtd daemon
-   to be running.
+   Using the box with the libvirt provider requires a correct Vagrant
+   installation on your machine, the plugin ``vagrant-libvirt`` to be
+   installed and the libvirtd daemon to be running.
```

Comment:
One idea: make all these points a bit more readable by using a list:

Using the box with the libvirt provider requires:

* a correct Vagrant installation on your machine,
* an installed plugin `vagrant-libvirt`, and
* a running libvirtd daemon.